src/Symfony/Component/HttpKernel/EventListener/ErrorListener.php

```diff
@@ -170,15 +166,40 @@ public static function getSubscribedEvents(): array
      */
     protected function logException(\Throwable $exception, string $message, string $logLevel = null): void
     {
+        if (null === $logLevel) {
+            $logLevel = $this->resolveLogLevel($exception);
+        }
+
         if (null !== $this->logger) {
```

Comment:
We should turn this into an early exit at the very beginning of the method. If no logger is set, determining the log level feels pointless.